spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -58,5 +58,56 @@
         )
       end
     end
+
+    it "gets the title tesim" do
+      expect(result["title_teim"]).to include(
+        "Harold B. Hoskins Papers"
+      )
+      expect(result["title_teim"]).to eq(
+        result["title_ssm"]
+      )
+    end
+
+    it "asserts that title filing si field is missing" do
+      expect(result["title_filing_si"]).to be_nil
+    end
+
+    context "YearRange normalizer tests" do
```

Comment:
Usually a context name starts with "when" so this could be "when there is a year range"